### PR TITLE
[admin] (Preview APIs) Switching beta types from the CDN to DT

### PR DESCRIPTION
**Pull request by AlexJerabek**

### CODE REVIEW

File: config/build.ts [hunk 1]
```diff
@@ -234,9 +234,10 @@ async function processSnippets(processedSnippets: Dictionary<SnippetProcessedDat
         const officeDtsReferences =
             snippet.libraries.split('\n')
                 .map(reference => reference.trim())
-                .filter(reference => reference.match(/.*((@types\/office-js)|(office\.d\.ts))$/gi));
+                .filter(reference => reference.match(/.*((@types\/office-js(-preview)?)|(office\.d\.ts))$/gi));
```
Comment (Zlatkovsky) [inline config/build.ts]: Do you want to add logic to enforce that the right d.ts is specified for the corresponding office.js reference -- just like we enforce office.js references to be of the canonical type?

Comment (Zlatkovsky) [reply config/build.ts]: What I'm thinking is just somewhere after here, for you to compare `officeJsReferences` and `officeDtsReferences` and ensure that if one is official or beta, than so is the other.

Comment (AlexJerabek) [reply config/build.ts]: I think we already do that on lines 269-271.

Comment (Zlatkovsky) [reply config/build.ts]: Oh, sweet.  I didn't look far enough down.  Sounds like you're all good, then.

**Pull request merged**